tests/spaceBasicTests.spec.ts

```diff
@@ -7,171 +7,229 @@ import { SpaceContextMenu } from "../page-objects/context-menus/spaceContextMenu
+      },
+    ],
   },
-  async ({ page }) => {
-    const leftMenu = new LeftMenu(page);
-    const createSpaceModal = new CreateSpaceModal(page);
-    const spaceContextMenu = new SpaceContextMenu(page);
-    const editSpaceNameModal = new EditSpaceNameModal(page);
-    const deleteSpaceModal = new DeleteSpaceModal(page);
-    const newSpaceName = "GUI TEST new space";
-    const renamedSpaceName = "RENAMED GUI TEST new space";
-
-    await page.goto("/");
-    await page.locator("cu-web-push-notification-banner").waitFor();
-
-    // Create new space
-    await leftMenu.clickOnElement("Create Space");
-    await createSpaceModal.typeSpaceName(newSpaceName);
-    await createSpaceModal.clickOnContinueButton();
-    await createSpaceModal.clickOnButton("Create Space");
-
-    await leftMenu.assertElementIsVisible(newSpaceName);
-
-    // Rename existing space
-    await leftMenu.rightClickOnElement(newSpaceName);
-    await spaceContextMenu.clickOnOption("Rename");
-    await editSpaceNameModal.typeSpaceName(renamedSpaceName);
-    await editSpaceNameModal.clickOnSaveButton();
-
-    await leftMenu.assertElementIsVisible(renamedSpaceName);
-
-    // Deleting renamed space
-    await leftMenu.rightClickOnElement(renamedSpaceName);
-    await spaceContextMenu.clickOnOption("Delete");
-    await deleteSpaceModal.typeSpaceName(renamedSpaceName);
-    await deleteSpaceModal.clickOnDeleteButton();
-    await deleteSpaceModal.waitForDeleting();
-
-    await leftMenu.assertElementIsNotVisible(renamedSpaceName);
-  },
+  async () => {
+    test(
+      "Basic End-2-End test for checking if creating, renaming and deleting existing space works correct",
+      {
+        tag: ["@space", "@e2e"],
+        annotation: {
+          type: "issue",
+          description: "https://github.com/marian51/poc-playwright-typescript-gui-api/issues/52",
+        },
+      },
+      async ({ page }) => {
+        const leftMenu = new LeftMenu(page);
+        const createSpaceModal = new CreateSpaceModal(page);
+        const spaceContextMenu = new SpaceContextMenu(page);
+        const editSpaceNameModal = new EditSpaceNameModal(page);
+        const deleteSpaceModal = new DeleteSpaceModal(page);
+        const newSpaceName = "GUI TEST new space";
+        const renamedSpaceName = "RENAMED GUI TEST new space";
+
+        await page.goto("/");
+        await page.locator("cu-web-push-notification-banner").waitFor();
+
+        // Create new space
+        await test.step("Step: Create new space", async () => {
+          await leftMenu.clickOnElement("Create Space");
+          await createSpaceModal.typeSpaceName(newSpaceName);
+          await createSpaceModal.clickOnContinueButton();
+          await createSpaceModal.clickOnButton("Create Space");
+
+          await leftMenu.assertElementIsVisible(newSpaceName);
+        });
+
+        // Rename existing space
+        await test.step("Step: Rename creating space", async () => {
```

Comment:
```suggestion
        await test.step("Step: Rename existing space", async () => {
```
IMO komentarze nad opisanymi stepami raczej do usunięcia